src/en/news/blog/2025/vllm-kv-caching/index.md

```diff
@@ -0,0 +1,622 @@
+systems we had an idea of what sort of system we would want to build to
+maximize throughput, while also drawing inspiration from choices made by major
+AI practitioners like Meta and OpenAI. Enter Meta’s contribution to the Open
+Compute project – the [Yosemite
+V3.5](https://www.opencompute.org/documents/yosemite-v3-5-platform-design-specification-v1-2-pdf) Sierra Point server platform. The YV3.5
+cubby occupies 3 OU and can be populated with 6x Sierra Point blades. Unlike
+conventional enterprise blade systems the YV3.5 platform does not have an
+integrated ethernet switch, instead each Sierra Point blade has OCP 3.0 slot
+for direct to host network connectivity. We wanted a system that was a spiritual
+successor to YV3.5 and Sierra Point, that reaped the advantages of cutting-edge
+processor designs and lithography. While surveying the server landscape across a
+whole host of OEMs there was one system that caught our attention, the
+Supermicro X14 2U 4-node GrandTwin Rear IO.
+
+![](images/smci-x14-grandtwin.png)
+
+[Supermicro X14 2U 4-node GrandTwin Rear
+IO](https://www.supermicro.com/en/products/system/datasheet/sys-212gt-hnr)
+
+Each node:
+•	1x Intel Xeon 6 6740E 96C/96T, 205W
+•	16x16GB DDR5-6400
+•	1x Broadcom 57608 2x200GbE
+•	6x 2.5” Kioxia CM6-R, 7.68TB Gen4 NVMe SSD
+•	RAID1 2x 480TB NVMe (boot) 
+
+This system is utilized to provide high-bandwidth all-flash object storage for
+the AI solution using IBM Storage Ceph 8.1.
+
+![](images/smci-gaudi3.png)
+
+[Supermicro Gaudi 3 AI Server
+SYS-822GA-NGR3](https://www.supermicro.com/en/products/system/datasheet/sys-822ga-ngr3)
+
+•	2x Intel Xeon 6 6960P 72C/144T
+•	24x 64GB DDR5-6400
+•	8x Gaudi 3 HL-325L accelerators
+•	Up to 8x 2.5" Gen5 NVMe SSD
+•	Scale-up networking: 21x 200GbE Gaudi NICs
+•	2x Broadcom 57608 1x400GbE
+
+This system is utilized to run inference workloads with the combination of vLLM
+and LMCache, leveraging Gaudi 3 accelerators from Intel. 
+
+![](images/smci-gpu-aplus.png)
+
+[Supermicro GPU A+ Server AS
+-8125GS-TNMR2](https://www.supermicro.com/en/products/system/datasheet/as-8125gs-tnmr2)
+
+•	1x AMD EPYC 9654 96C/192T
+•	24x 96GB DDR5-4800
+•	8x AMD MI300X accelerators
+•	Up to 8x 2.5" Gen5 NVMe SSD
+•	Scale-up networking: 4x400GbE
+•	Storage and GPU scale-out networking: 4x NVIDIA MT28908 ConnectX-6 200GbE
+
+This system is utilized to run inference workloads with the combination of vLLM
+and LMCache, leveraging MI300X accelerators from AMD.
+
+![](images/smci-sw.png)
+
+[SSE-T7132S - 400Gb Ethernet
+Switch](https://www.supermicro.com/en/products/accessories/Networking/SSE-T7132SR.php)
+
+•	32x QSFP-DD 400GbE, or 64x QSFP56 / 128x QSFP28 with breakout cables
+•	25.6Tb/s switching capacity
+•	SONiC OS
+•	RoCEv2/RDMA support with PFC
+
+For simplicity we used a single fixed-port 400Gb switch for both GPU-to-GPU and
+the storage fabric.
+
+## Host configuration
+
+* Performance profile set in BIOS
+* Set the tuned profile to network-latency
```

Comment:
TuneD

backticks around network-latency

Did you set thermal/power profiles in BIOS/BMC to performance vs balanced?  Not sure if these SMCI systems have those; with Dells they matter.